example/src/ComponentDemo.react.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import * as React from "react";
+import cn from "classnames";
+import { Button } from "tabler-react";
+import SyntaxHighlighter from "react-syntax-highlighter/prism";
+import { prism } from "react-syntax-highlighter/styles/prism";
+import jsxToString from "jsx-to-string";
+
+type Props = {|
+  +children?: React.Node,
+  +className?: string,
+|};
+
+type State = {|
+  codeOpen: boolean,
+|};
+
+class ComponentDemo extends React.PureComponent<Props, State> {
+  state = {
+    codeOpen: false,
+  };
+  handleSourceButtonOnClick = (e: SyntheticMouseEvent<HTMLLinkElement>) => {
+    e.preventDefault();
+    this.setState(s => ({ codeOpen: !s.codeOpen }));
+  };
+
+  render() {
+    const { className, children } = this.props;
+    const { codeOpen } = this.state;
+    const classes = cn(`highlight`, className);
+    return (
+      <div style={{ position: "relative", marginBottom: 20 }}>
```

Comment:
Let's try to avoid writing inline styles.  Can we make `ComponentDemo.css` instead and import that at the top of this file and just apply the classnames?